Misc/NEWS.d/next/Library/2021-04-03-18-03-44.bpo-43723.uBhBZS.rst

```diff
@@ -0,0 +1,10 @@
+The following `threading` methods are now deprecated and should be replaced:
+`threading.currentThread` => :func:`threading.current_thread`;
+`threading.activeCount` => :func:`threading.active_count`;
+`threading.Condition.notifyAll` => :meth:`threading.Condition.notify_all`;
+`threading.Event.isSet` => :meth:`threading.Event.is_set`;
+`threading.Thread.setName` => :attr:`threading.Thread.name`;
+`threading.thread.getName` => :attr:`threading.Thread.name`;
+`threading.Thread.isDaemon` => :attr:`threading.Thread.daemon`;
+`threading.Thread.setDaemon` => :attr:`threading.Thread.daemon`.
+Patch by Jelle Zijlstra.
```

Comment:
```suggestion
The following `threading` methods are now deprecated and should be replaced:

* `threading.currentThread` => :func:`threading.current_thread`;
* `threading.activeCount` => :func:`threading.active_count`;
* `threading.Condition.notifyAll` => :meth:`threading.Condition.notify_all`;
* `threading.Event.isSet` => :meth:`threading.Event.is_set`;
* `threading.Thread.setName` => :attr:`threading.Thread.name`;
* `threading.thread.getName` => :attr:`threading.Thread.name`;
* `threading.Thread.isDaemon` => :attr:`threading.Thread.daemon`;
* `threading.Thread.setDaemon` => :attr:`threading.Thread.daemon`.

Patch by Jelle Zijlstra.
```

Reply:
I had to do it a little differently to get the markup working and work around python/core-workflow#394.

Reply:
> I had to do it a little differently to get the markup working and work around python/core-workflow#394.

If you put each item on a long line, you should not be affected by https://github.com/python/core-workflow/issues/394 no?

Reply:
blurb reflows the lines to make them fit into 76 characters, so I have to make each line fit into 76 characters to avoid the bug.